asteroid/engine/system.py

```diff
@@ -114,18 +114,11 @@ def training_step(self, batch, batch_nb):
         tensorboard_logs = {"train_loss": loss}
         return {"loss": loss, "log": tensorboard_logs}
 
-    def optimizer_step(
-        self,
-        epoch: int,
-        batch_idx: int,
-        optimizer: Optimizer,
-        optimizer_idx: int,
-        second_order_closure: Optional[Callable] = None,
-    ) -> None:
+    def optimizer_step(self, *args, **kwargs):
         for sched in self.scheduler:
```

Comment:
If think this misses a `if self.scheduler is not None:`

Reply:
You're right, thanks

Reply:
It's also missing in the current code then, which was just released haha. 
I'll see if it breaks (probably) and release 0.3.2 then.